### PR TITLE
Clean up code so it passes validation

### PR DESCRIPTION
**Pull request by demisx**
Hi. I've cleaned up your code a bit, so it passes JSHint validation.

### CODE REVIEW

File: spec/paginate-anything.spec.js [hunk 1]
```diff
@@ -38,25 +36,25 @@ server.listen(3000);
...
+ */
 function paginatedRequest(total_items, max_range_size, range, callback)
 {
 	if (total_items >= Infinity)
 	{
 		total_items = '*';
 	}
-    
-    if (range !== null) {
-        var headers = {
```
Comment (demisx) [inline spec/paginate-anything.spec.js]: Javascript has only function scope. No need to redefine var inside a block.

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "src/paginate-anything.js",
   "private": false,
   "scripts": {
-	"test": "jasmine-node spec/"
+	"test": "node_modules/jasmine-node/bin/jasmine-node spec"
```
Comment (demisx) [inline package.json]: Not everyone has jasmine-node installed globally. This will allow `npm test` to run.

File: .jshintrc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (demisx) [inline .jshintrc]: Consider using JSHint in your editor and grunt tasks.

**Comment by polo2ro** [comment]
good work!

**Pull request merged**